1208-projects/project_wizard.feature

```diff
@@ -0,0 +1,166 @@
+        And they can see User search field
+        Examples:
+            | Role                  |
+            | Curator               |
+            | Project Owner         |
+            | Project Manager       |
+            | Local Project Manager |
+    
+    @2906c
+    Scenario: User adds a Project Participant
+        Given User opens Dialog for adding Project Participant
+        When they enter a name in the User search field
+        And they select a role
+        And they select a User from the search
+        And they click the Add Button
+        Then they see the User listed as a Project Participant
+
+    @2907a
+    Scenario Outline: Privileged user opens Participants tab for Project
+        Given User opens Participants tab for Project
+        When User has role "<Role>" in the project
+        Then they see Button Add Project Manager
+        Examples:
+            | Role            |
+            | Curator         |
+            | Project Owner   |
+            | Project Manager |
+
+
+    @2907b
+    Scenario Outline: Privileged user opens Dialog for adding Project Manager
+        Given Privileged user opens Participants tab for Project
+        And User has role "<Role>" in the project:
+        When they click the Add Project Manager Button
+        Then they see fields:
+            | Start Date  |
+            | User search |
+        Examples:
+            | Role            |
+            | Curator         |
+            | Project Owner   |
+            | Project Manager |
+
+    @2907c
+    Scenario: Privileged user adds a Project Manager
+        Given Privileged user opens Dialog for adding Project Manager
+        When they select a Start Date
+        And they enter a name in the User search field
+        And they select a User from the search
+        And they click the Add Button
+        Then they see the User listed as a Project Manager
+
+    @2908a
+    Scenario Outline: User opens Financing tab for Project
+        Given User sees the Project Wizard
+        And User has the "<Role>" role in the project
+        When they click the Financing tab
+        Then they see fields for:
+            | Financing Code |
+        And the fields are "<FieldStatus>"
+        And they see a Previous Button
+        And they see a Support Button
+        And they see a Save and Present Button
+        Examples:
+            | Role                  | FieldStatus |
+            | Curator               | Enabled     |
+            | Project Owner         | Enabled     |
+            | Project Manager       | Enabled     |
+            | Local Project Manager | Disabled    |
+
+    @2908b
+    Scenario: User adds a Financing source for Project
+        Given User opens Financing tab for Project
+        When they enter a search text in the search field
+        And they select a Financing source from the search results
+        Then they see that the Financing Code is added to the project
```

Comment:
hva skal det søkes mot her? er det noe som eksiterer, eller må det utvikles nytt endpoint?

Reply:
Der er mange ulike kilder som kan søkes i, som ein start bør vi i det minste tilby
* https://prosjektbanken.forskningsradet.no/prosjektbanken/rest/cristin/search?query=111
* https://beta.explore.openaire.eu/search/advanced/projects?q=&op=and

Eg ønsker å tilby eit enkelt søk, der kilden er underordnet treffet. Slik at forskeren kan søke uten å tenke på å velge rette kilde - før evt. i trefflisten, der kan gjerne kilden komme fram. Dette er så vidt diskutert med Ru. Eg har starta ein dialog med Helse Vest om kilde for helseprosjekter, det blir nok den tredje. Eg har ikkje noko API enda for den.

![image](https://user-images.githubusercontent.com/11860616/124908076-338a2e80-dfe9-11eb-8a09-408f5151aca4.png)
Som du ser av denne finansieringen frå EU, så pålegger den bruk av ein Open Access lisens. Slikt skal vi sjå på etterkvart.

Reply:
Det er vert å nemne at alle finansieringskilder (stort sett) omtaler sine finansieringer som prosjekt. Slike prosjekt er i vår datamodell ein finansiering (GrantID) og eit prosjekt kan ha mange slike kilder til finansiering.